cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -47,87 +50,133 @@ <h1>{{ page.title }}</h1>
                 {% endif %}
                 ${{ '{:,.0f}'.format( page.salary_min ) }}–${{ '{:,.0f}'.format( page.salary_max ) }}
                 <a class="a-link 
-                          a-link__jump 
-                          a-link__icon 
                           cf-icon 
                           cf-icon__after 
                           cf-icon-download 
                           pay-grade-link"
                    href="http://files.consumerfinance.gov/f/documents/cfpb_pay_scales.pdf">
-                    <span class="a-link_text">See information on grades and base pay ranges</span>
+                   See information on grades and base pay ranges
                 </a>
             </dd>
-                
+            {% if usajobs_links %}
+                <dt>
+                    Who may apply:
+                </dt>
+                <dd>
+                     {% for usajobs_link in usajobs_links %}
+                        {% set applicant_type = usajobs_link.applicant_type %}
+                        {{ applicant_type.display_title if applicant_type.display_title else applicant_type }}{{',' if not loop.last else ''}}
+                        {% endfor %}
+                </dd>
+            {% endif %}
         </dl>
     </section>
 
     <section class="block
                     block__padded-top
-                    block__border-top">
-        <h2>Job Description</h2>
-        {{ parse_links( page.description ) | safe }}
-    </section>
+                    block__border-top
+                    block__flush-bottom
+                    vacancy-announcement_description">
+        <div data-qa-hook="expandable"
+             data-read-more="true"
+             class="o-expandable
+                    o-expandable__read-more">
+            <div class="o-expandable_content" id="job-description-expandable">
+                <div class="o-expandable_content-animated">  
+                    <h4>Summary</h4>
+                    {{ parse_links( page.description ) | safe }}
+                    {% if page.responsibilities %}
+                        <div class="block block__sub">
+                            <h4>Responsibilities</h4>
+                            {{ page.responsibilities | safe }}
```

Comment:
This output should be wrapped in `parse_links()` like `page.description` is above. Same for `travel_details` and `additional_section_content`.

Reply:
Good catch, thanks!